modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -146,11 +148,24 @@ private void _validate(ObjectEntry objectEntry) {
 					_objectFieldLocalService.getObjectField(
 						objectEntry.getObjectDefinitionId(), "path");
 
+				String message = "";
+				String messageKey = "";
+
+				if (pathString.startsWith(pathString)) {
```

Comment:
I think it is not correct! Maybe 

`pathString.startsWith("/")` or `pathString.startsWith(StringPool.SLASH)`